hystrix/src/test/java/feign/hystrix/HystrixBuilderTest.java

```diff
@@ -148,7 +148,7 @@ public void fallbacksApplyOnError() {
   @Test
   public void errorInFallbackHasExpectedBehavior() {
     thrown.expect(HystrixRuntimeException.class);
-    thrown.expectMessage("contributors failed and fallback failed.");
+    thrown.expectMessage("feign#public abstract java.util.List feign.hystrix.HystrixBuilderTest$GitHub.contributors(java.lang.String,java.lang.String) failed and fallback failed.");
```

Comment:
So it is not ideal, but in my opinion this is a separate issue from the one we are trying to solve 
![image](https://cloud.githubusercontent.com/assets/524254/17714024/de6541cc-63cb-11e6-8e4a-4a6a83b2e19b.png)